### DISCUSSION TITLE
Dependent relationship option pickers in a form

### DISCUSSION THREAD

**Discussion by egattulli** [Budibase/budibase#10494] [Help] [answered]
Hi everyone!

I need to create a form in which are contained five relationship option pickers. The last three of them are dependent from each other. Here are the fields: 1. Production Phase; 2. Production Subphase; 3. Production Subphase Group. I would like that users would select first the Production Phase and the Production Subphase options would be filtered on the basis of their choice. The same should happen with the Production Subphase Group. So, the Production Subphase  Group options should be filtered through the first two relationship option pickers.

Currently, I need this form to create rows in the referenced table. Once I understand how to achieve the result I want, I will create another form to update or delete rows.

Thanks in advance for your help!

**Comment by melohagan** [comment, marked as answer]
Hey @egattulli 

Sorry for the late response, this one slipped past me! 

I believe this docs page highlights how to do something similar to what you want: https://docs.budibase.com/docs/filter-by-relationships
